src/lib.rs

```diff
@@ -725,10 +728,17 @@ pub fn mount(mount_point: &Path, options: &[&str], mappings: &[Mapping], ttl: Ti
     };
 
     let config_handler = {
-        let mut input = concurrent::ShareableFile::from(input);
-        let reader = io::BufReader::new(input.reader()?);
-        let writer = io::BufWriter::new(output);
+        let mut input = match input {
+            Some(file) => concurrent::ShareableFile::from(file),
+            None => concurrent::ShareableFile::from_stdin(io::stdin()),
+        };
+        let reader = input.reader()?;
+
         let handler = thread::spawn(move || {
+            let writer: Box<io::Write> = match output {
+                Some(file) => Box::from(file),
+                None => Box::from(io::stdout()),
```

Comment:
Please lock it. Otherwise you engage the lock every time you try to write to it.

Reply:
Stale after rewrite per the comment above.